modmesh/pilot/_euler1d.py

```diff
@@ -815,44 +907,78 @@ def setup(self):
         self.plot_config = self.app.plot_config
 
     def on_open(self):
-        self.state.plot_config_open = True
+        self.plot_config_open = True
+        if not self.plot_config_dialog:
+            self.plot_config_dialog = PlotConfigDialog(self.app, self)
+        self.plot_config_dialog.exec_()
 
     def on_close(self):
-        self.state.plot_config_open = False
+        self.plot_config_open = False
+        if self.plot_config_dialog:
+            self.plot_config_dialog.close()
 
-    def content(self):
+    def init_UI(self):
```

Comment:
It seems that we mixed `snake_case` and `camelCase` naming convention in this file.  Let's fix it later.